src/pytestqt/qtbot.py

```diff
@@ -623,7 +623,7 @@ def keySequence(widget, key_sequence):
 
     @staticmethod
     def keyToAscii(key):
-        if qt_api.pytest_qt_api == "pyqt5":
+        if not hasattr(qt_api.QtTest.QTest, "keyToAscii"):
```

Comment:
Seems like the better check to me, given that `keyToAscii` apparently wasn't available with PySide6 6.0.0 (otherwise the test would've failed in #330 already).